aesara/scan/op.py

```diff
@@ -3427,7 +3427,7 @@ def profile_printer(
             )
 
 
-@op_debug_information.register(Scan)  # type: ignore[has-type]
+@op_debug_information.register(Scan)
```

Comment:
Oh yikes, that looks tricky! Thanks for the context. I will need to follow up here to try and understand better what's going on.

Reply:
Yeah, I would _really_ like to fix that problem at the source, but it was/is annoyingly elusive.

Reply:
Working around this with `warn_unused_ignores = false` as below, but I'll still take a stab at understanding what's really going on.

Reply:
Unfortunately I haven't gotten a chance to look into this yet. The issue no longer creeps up here since readding `warn_unused_ignores = false` in #1376. Is this blocking the merge?